torch/_dynamo/output_graph.py

```diff
@@ -1366,9 +1370,30 @@ def add_output_instructions(self, prefix: List[Instruction]) -> None:
         self.output_instructions.extend(prefix)
         self.should_exit = True
 
-    def install_global(self, name, value) -> None:
+    def install_global_unsafe(self, name, value) -> None:
+        """
+        WARNING: prefer the safer `install_global_once`.
+        torch.compile instances should be independent of each other;
+        one footgun is to have one instance depend on the existence of
+        a global installed by another instance. This can happen if we mangle
+        a global the same way across both instances.
+        """
         self.cleanups.append(CleanupHook.create(self.global_scope, name, value))
```

Comment:
So is the idea we're going to eventually switch the rest of the sites over to this too?

Reply:
If we can, but they're difficult to refactor :/